### ISSUE TITLE
Not displayed after Alert or ActionSheet

### ISSUE THREAD

**Issue by timbrand** [jamesmontemagno/MediaPlugin#169]
## Bug 
When an Alert of ActionSheet is displayed right before the PickPhotoAsync() or TakePhotoAsync() is called, the window is not displayed/visible.

Version Number of Plugin:
Device Tested On: iOS (device and simulator)
Simulator Tested On: multiple iOS simulators

### Expected Behavior
Should show the photo-take view or image-picker;

### Actual Behavior
Displayed nothing. When calling the method again, an exception is throws of a multiple instance;

### Steps to reproduce the Behavior
(Xamarin.Forms) add:
`await DisplayAlert("alert", "message", "cancel");`
before the PickPhoto or TakePhoto call.

When adding a delay of 100ms after the Alert or ActionSheet, it's working fine. Which is my current workaround;

**Comment by hareshkainth** [comment]
Hi, I get this same problem. I tried the delay but nothing happens on first attempt. When I try the Gallery or Camera, I get an exception. 

Media Plugin: 2.6.2 (PCL + DROID + IOS)

Im rolling back to version 2.6.0 to see if I can recreate the problem or not.

**Comment by Hiilbig** [comment]
I experience the same issue with Media Plugin v 2.6.3 (PLC + iOS 10). The workaround of a delay after Alert does not work for me. Any new ideas?

**Comment by hareshkainth** [comment]
I have the same problem now. I don't understand why I'm get this issue.

**Comment by hareshkainth** [comment]
I just updated my Xamarin Forms and Forms.Map package to 2.3.4.192-pre2 and everything started to work. Need to test everything again to make sure nothing is broken but happy days again.

**Comment by jamesmontemagno** [comment]
Did not have an issue in 2.3.2 or 2.3.3 of xamarin.forms and latest package. Tested on iPad Pro and iphone 6. Could be some weird timing thing with the dialog box.


This was my code:

```
await DisplayAlert("Hello", "World", "Ok");
            var file = await CrossMedia.Current.TakePhotoAsync(new StoreCameraMediaOptions
            {
                Directory = "InventoryManagement",
                Name = "item.jpg",
                PhotoSize = PhotoSize.Medium,
                CompressionQuality = 42,
                SaveToAlbum = false,
                DefaultCamera = CameraDevice.Rear,

            });
```

**Comment by nonlin** [comment]
I'd say this is still an issue. I do the same except I use the Action Sheet's result to determine which picker to use, so only extra thing I have is a few if/else statements after the action sheet and none of the picker sheets  open.
 
I've fixed it so far by wrapping the ActionSheet into a Task that I can cancel once a choice has been made on the action sheet and then the pickers works no problem.

**Comment by pbhesser3** [comment]
We've ran into this issue as well. Is the reason why it is closed is due to not seeing the issue in 2.3.4.192-pre2?  If so when will 2.3.4.192-pre2 no longer be a pre-release?
